src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -550,7 +550,7 @@ Public Class BuildDevDivInsertionFiles
                                                                 packageName,
                                                                 packageVersion,
                                                                 isNative:=native IsNot Nothing,
-                                                                isFacade:=frameworkAssemblies IsNot Nothing))
+                                                                isFacade:=frameworkAssemblies IsNot Nothing OrElse packageName = "System.IO.Pipes.AccessControl"))
```

Comment:
curious why this is special?

Reply:
@jasonmalinowski Can you clarify here?

Reply:
@heejaechang It's not, the condition should more generally be rewritten. This is a tactical fix.

Reply:
One of the happiest days of my future self is deleting this file.